specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/examples/bookmarks/relations/CreateBookmarkRelation.json

```diff
@@ -0,0 +1,57 @@
+{
+    "parameters": {
+      "api-version": "2019-01-01-preview",
+      "subscriptionId": "d0cfe6b2-9ac0-4464-9919-dccaee2e48c0",
+      "resourceGroupName": "myRg",
+      "workspaceName": "myWorkspace",
+      "operationalInsightsResourceProvider": "Microsoft.OperationalInsights",
+      "bookmarkId": "2216d0e1-91e3-4902-89fd-d2df8c535096",
+      "relationName": "4bb36b7b-26ff-4d1c-9cbe-0d8ab3da0014",
+      "relationInputModel": {
+        "properties": {
+            "sourceRelationNode": {
+                "relationNodeId": "afbd324f-6c48-459c-8710-8d1e1cd03812",
+                "relationNodeKind": "Case"
+            },
+            "targetRelationNode": {
+                "relationNodeId": "2216d0e1-91e3-4902-89fd-d2df8c535096",
+                "relationNodeKind": "Bookmark"
+            }
+        }
+      }
```

Comment:
PUT body should not differ drastically from PUT response or the GET response.

Reply:
This applies to the case relations as well.

---
In reply to: [324920170](https://github.com/Azure/azure-rest-api-specs/pull/7191#discussion_r324920170) [](ancestors = 324920170)